app/models/comment.php

```diff
@@ -20,17 +20,116 @@ public static function getComments($offset, $limit, $id)
         $db = DB::conn();
         $rows = $db->rows("SELECT * FROM comment 
                         WHERE thread_id = ? ORDER BY created ASC LIMIT {$offset}, {$limit}", array($id));
```

Comment:
but its still a public method po. i can still call this on other controllers like
```php
Comment::getCommentById("a", "b", "c");
```

and also, please move the most important parameter on the left side. you are getting array of object from id. so i think it must be 
```php
public static function getCommentById($id, $limit, $offset);
```

parameters that can have default value or can be null must be placed on the right side :bow: